src/Raven.Server/Documents/TcpHandlers/SubscriptionDocumentsFetcher.cs

```diff
@@ -77,7 +78,11 @@ public class SubscriptionDocumentsFetcher
              IncludeDocumentsCommand includesCmd,
             long startEtag)
         {
-            int size = 0;
+            int numberOfDocs = 0;
+            Size size = new Size(0 , SizeUnit.Megabytes);
+            _MaximumAllowedMemory = new Size((PlatformDetails.Is32Bits ||
```

Comment:
This should be initialized once, not each time

Reply:
done